docs/howtos/layers/tracks.md

```diff
@@ -11,21 +11,58 @@ kernelspec:
   name: python3
 ---
 
-# Using the tracks layer
+# Using the `tracks` layer
 
-In this document, you will learn about the `napari` `Tracks` layer, including
+In this document, you will learn about the `napari` `tracks` layer, including
 displaying tracks and defining relationships between tracks.
 
-## When to use the tracks layer
+For more information about layers, refer to [Layers at a glance](../../guides/layers).
 
-The tracks layer allows you to display trajectories in `nD+t` while visualizing
-the recent history of the track via a fading tail.
+## When to use the `tracks` layer
+
+The `tracks` layer allows you to display trajectories in `nD+t` while
+visualizing the recent history of the track via a fading tail.
 
 Each track can have annotations associated with it using the `Tracks.properties`
 dictionary. These properties can be used to set the colors of the tracks.
 
 For example, when displaying tracks of different classes/types, one could
-automatically set color the individual tracks by their respective class/type.
+automatically set the color of the individual tracks by their respective
+class/type.
+
+Tracks layers can be created only programmatically, i.e. in the console, using a
+script, etc. Please refer to [A simple example](#a-simple-example) and use the
```

Comment:
I think the warning about being created only programmatically is very important and should be provided earlier and more emphatically!

Reply:
I agree, Peter.  Plainly stated is better in tech writing.